tools/test/generateAllScreenshots.py

```diff
@@ -132,7 +132,10 @@ def generateJavascriptFile():
             simpleFile = file[:3] + "T" + file[4:-7] + l + file[-5:-4]
             translatedFile = "./screenshots/" + l + "/" + simpleFile + ".png"
             if os.path.exists(translatedFile):
-                dataForFile.append(1)
+                # Get the last modified date of the file in seconds and round to days
+                date = os.popen("git log -1 --format=%ct -- \"" + translatedFile + "\"").read().strip()
```

Comment:
This operation is taking time and regarding the number of files, it's a bit annoying. But will be run by the CI, so I guess it's fine...